src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -129,22 +133,58 @@ public static void logOnLowLevel(String command, String description, boolean suc
     log(command, description, success, true);
   }
 
+  //TODO: Check error class here
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)